### PR TITLE
Show flashcards

### PR DESCRIPTION
**Pull request by groundberry**

### CODE REVIEW

File: src/Flashcard.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+
```
Comment (unindented) [inline src/Flashcard.css:11]: Unnecessary empty line here.

File: src/Sidebar.js [hunk 2]
```diff
@@ -14,7 +14,11 @@ class Sidebar extends Component {
             My tags
           </h2>
           <div className="col-sm-8 col-xs-12 col-sm-offset-2">
-            <Tags tags={this.props.tags}/>
+            <Tags
+              tags={this.props.tags}
+              onClickTag={this.props.onClickTag}
+              flashcards={this.props.flashcards}
```
Comment (unindented) [inline src/Sidebar.js]: Why does the `Tags` component need the array of flashcards?

**Review by unindented — CHANGES_REQUESTED** [review]

File: src/Flashcard.css (hunk 1 above)
Comment (groundberry) [reply src/Flashcard.css:11]: Fixed

File: src/Sidebar.js (hunk 2 above)
Comment (groundberry) [reply src/Sidebar.js]: Fixed

**Review by unindented — APPROVED** [review]

**Pull request merged**